hop/list_topics.py

```diff
@@ -45,3 +49,9 @@ def _main(args):
 
 def _add_parser_args(parser):
     cli.add_client_opts(parser)
+
```

Comment:
Since this is just listing the possible topics the user has access to, is passing in the group ID ever needed?

Reply:
It is not required by Kafka, and in fact I suspect doesn't even do anything. Unfortunately, Confluent Kafka refuses to create a consumer object without a group: https://github.com/confluentinc/confluent-kafka-python/blob/b7f8dce998ec254f54c15e514850d7404c6a71a3/src/confluent_kafka/src/confluent_kafka.c#L1851

Reply:
Just realized I didn't highlight the correct lines. Oh yeah, I understand that about the group ID in terms of the confluent client requiring this. I wonder if we need to expose this to the user though, since any group ID should supposedly return the same results and we could just generate a random group ID to satisfy that.

Reply:
Good point, I don't think there's a need to expose this, so I'll remove the user-facing option.